### ISSUE TITLE
What extra step need to run rake db:reset on local machine?

### ISSUE THREAD

**Issue by bao-qian** [jtreanor/SummerCourses#49]
@jtreanor 
I can't run rake db:reset on my local machine

``` bash
-- assume_migrated_upto_version(20130306165555, ["/Users/death/Documents/SummerCourses/db/migrate"])
   -> 0.0008s
rake aborted!
Connection refused - connect(2)
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/mail-2.4.4/lib/mail/network/delivery_methods/smtp.rb:144:in `deliver!'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/mail-2.4.4/lib/mail/message.rb:2034:in `do_delivery'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/mail-2.4.4/lib/mail/message.rb:229:in `block in deliver'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/actionmailer-3.2.12/lib/action_mailer/base.rb:415:in `block in deliver_mail'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activesupport-3.2.12/lib/active_support/notifications.rb:123:in `block in instrument'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activesupport-3.2.12/lib/active_support/notifications/instrumenter.rb:20:in `instrument'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activesupport-3.2.12/lib/active_support/notifications.rb:123:in `instrument'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/actionmailer-3.2.12/lib/action_mailer/base.rb:413:in `deliver_mail'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/mail-2.4.4/lib/mail/message.rb:229:in `deliver'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/devise-2.2.3/lib/devise/models/authenticatable.rb:159:in `send_devise_notification'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/devise-2.2.3/lib/devise/models/recoverable.rb:48:in `send_reset_password_instructions'
/Users/death/Documents/SummerCourses/app/models/admin_user.rb:37:in `block in <class:AdminUser>'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activesupport-3.2.12/lib/active_support/callbacks.rb:409:in `_run__2639066868369867524__create__3655043815315807516__callbacks'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activesupport-3.2.12/lib/active_support/callbacks.rb:405:in `__run_callback'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activesupport-3.2.12/lib/active_support/callbacks.rb:385:in `_run_create_callbacks'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activesupport-3.2.12/lib/active_support/callbacks.rb:81:in `run_callbacks'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/callbacks.rb:268:in `create'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/persistence.rb:348:in `create_or_update'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/callbacks.rb:264:in `block in create_or_update'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activesupport-3.2.12/lib/active_support/callbacks.rb:414:in `_run__2639066868369867524__save__3655043815315807516__callbacks'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activesupport-3.2.12/lib/active_support/callbacks.rb:405:in `__run_callback'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activesupport-3.2.12/lib/active_support/callbacks.rb:385:in `_run_save_callbacks'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activesupport-3.2.12/lib/active_support/callbacks.rb:81:in `run_callbacks'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/callbacks.rb:264:in `create_or_update'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/persistence.rb:84:in `save'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/validations.rb:50:in `save'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/attribute_methods/dirty.rb:22:in `save'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/transactions.rb:259:in `block (2 levels) in save'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/transactions.rb:313:in `block in with_transaction_returning_status'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/connection_adapters/abstract/database_statements.rb:192:in `transaction'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/transactions.rb:208:in `transaction'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/transactions.rb:311:in `with_transaction_returning_status'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/transactions.rb:259:in `block in save'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/transactions.rb:270:in `rollback_active_record_state!'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/transactions.rb:258:in `save'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/persistence.rb:45:in `create'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/persistence.rb:42:in `block in create'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/persistence.rb:42:in `collect'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/persistence.rb:42:in `create'
/Users/death/Documents/SummerCourses/db/seeds.rb:270:in `<top (required)>'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activesupport-3.2.12/lib/active_support/dependencies.rb:245:in `load'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activesupport-3.2.12/lib/active_support/dependencies.rb:245:in `block in load'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activesupport-3.2.12/lib/active_support/dependencies.rb:236:in `load_dependency'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activesupport-3.2.12/lib/active_support/dependencies.rb:245:in `load'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/railties-3.2.12/lib/rails/engine.rb:520:in `load_seed'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/railties/databases.rake:333:in `block (2 levels) in <top (required)>'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/gems/activerecord-3.2.12/lib/active_record/railties/databases.rake:276:in `block (2 levels) in <top (required)>'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/bin/ruby_noexec_wrapper:14:in `eval'
/Users/death/.rvm/gems/ruby-1.9.3-p385@rails3tutorial2ndEd/bin/ruby_noexec_wrapper:14:in `<main>'
Tasks: TOP => db:setup => db:seed
(See full trace by running task with --trace)
```

**Comment by jtreanor** [comment]
Sorry about this. The reason for that is that it fails to to send an email on your local machine. This is fixed in jtreanor/SummerCourses@5061c3099db5202151c32137d2f286eb8fd3face

**Comment by bao-qian** [comment]
Thanks.
